datashare-app/src/main/java/org/icij/datashare/tasks/TaskManagerAmqp.java

```diff
@@ -17,6 +17,7 @@ public TaskManagerAmqp(AmqpInterlocutor amqp, TaskRepository taskRepository, Pro
 
     TaskManagerAmqp(AmqpInterlocutor amqp, TaskRepository taskRepository, PropertiesProvider propertiesProvider,
                     Runnable eventCallback) throws IOException {
-        super(amqp, taskRepository, Utils.getRoutingStrategy(propertiesProvider), eventCallback);
+        super(amqp, taskRepository, Utils.getRoutingStrategy(propertiesProvider), eventCallback,
+                Integer.parseInt(propertiesProvider.get("taskManagerPollingInterval").orElse(String.valueOf(DEFAULT_TASK_POLLING_INTERVAL))));
```

Comment:
I would include the unit in the variable names `taskManagerPollingIntervalMilliseconds` and `DEFAULT_TASK_POLLING_INTERVAL_MS` to be consistent with the DatashareCLI naming